src/store/ChartState.js

```diff
@@ -259,51 +259,50 @@ class ChartState {
             this.rootNode = this.mainStore.chart.rootNode;
         }
         this.rootNode.querySelector('.chartContainer').style.backgroundColor = Theme[`${theme}_chart_bg`];
-        // change chart colors to grey if the current market is closed and it is not a static chart
+        // reduce opacity to 32% if the current market is closed and it is not a static chart
         if (isChartClosed && !this.isStaticChart) {
-            const closedChartColor = 'rgba(129, 133, 152, 0.35)';
-            this.stxx.setStyle('stx_mountain_chart', 'borderTopColor', closedChartColor);
+            this.stxx.setStyle('stx_mountain_chart', 'borderTopColor', Theme[`${theme}_chart_mountain_border_closed`]);
             this.stxx.setStyle('stx_mountain_chart', 'backgroundColor', Theme[`${theme}_chart_mountain_bg`]);
             this.stxx.setStyle('stx_mountain_chart', 'color', Theme[`${theme}_chart_mountain_bg_shade`]);
```

Comment:
Also, please manage with Dervaj, to see, if we should apply opacity on `Area` shade area, its not so obvious in light theme, but dark theme